kubeadm/scripts/Install-Containerd.ps1

```diff
@@ -77,21 +77,19 @@ $env:Path += ";$global:ConainterDPath"
 containerd.exe config default | Out-File "$global:ConainterDPath\config.toml" -Encoding ascii
 #config file fixups
 $config = Get-Content "$global:ConainterDPath\config.toml"
-$config = $config -replace "bin_dir = (.)*$", "bin_dir = `"c:/opt/cni/bin`""
-$config = $config -replace "conf_dir = (.)*$", "conf_dir = `"c:/etc/cni/net.d`""
+$config = $config -replace "bin_dir = (.)*$", "bin_dir = `"C:\\Program Files\\containerd\\cni\\bin`""
+$config = $config -replace "conf_dir = (.)*$", "conf_dir = `"C:\\Program Files\\containerd\\cni\\conf`""
 $config | Set-Content "$global:ConainterDPath\config.toml" -Force 
 
-mkdir -Force c:\opt\cni\bin | Out-Null
-mkdir -Force c:\etc\cni\net.d | Out-Null
-
 Write-Output "Getting SDN CNI binaries"
-DownloadFile "c:\opt\cni\cni-plugins.zip" https://github.com/microsoft/windows-container-networking/releases/download/v0.2.0/windows-container-networking-cni-amd64-v0.2.0.zip
-Expand-Archive -Path "c:\opt\cni\cni-plugins.zip" -DestinationPath "c:\opt\cni\bin" -Force
+mkdir -Force $global:ConainterDPath\cni | Out-Null
+DownloadFile "$global:ConainterDPath\cni\cni-plugins.zip" https://github.com/microsoft/windows-container-networking/releases/download/v0.2.0/windows-container-networking-cni-amd64-v0.2.0.zip
```

Comment:
With the hostprocess images, these binaries are ship as part of the installer for the image, we shouldn't need to install them at all.

Reply:
https://github.com/kubernetes-sigs/sig-windows-tools/blob/6d113824ba7bda0b5c449883c4fc3ac97fa9ea56/hostprocess/calico/install/Dockerfile.install#L14-L15

https://github.com/kubernetes-sigs/sig-windows-tools/blob/6d113824ba7bda0b5c449883c4fc3ac97fa9ea56/hostprocess/flannel/flanneld/Dockerfile#L14-L19